page_groups.php

```diff
@@ -59,6 +64,31 @@
 
     $tabs = new tabs($url);
     echo $output->render_tabs($tabs);
+
+    if (empty($month)) {
+        $month = page_group::get_current_month();
+        $button = null;
+    } else {
+        $button = $output->render(new \single_button($url, get_string('to_current_month', 'tool_excimer')));
+    }
+
+    $prevmonth = $month - 1;
+    if ($prevmonth % 100 == 0) {
+        $prevmonth -= 88; // Subtract 1 year, but add 12 months.
+    }
+    $nextmonth = $month + 1;
+    if ($nextmonth % 100 == 13) {
+        $nextmonth += 88; // Add one year, but subract 12 months.
+    }
```

Comment:
There are a few magic numbers in the mix here. Could we add a description or set them to a variable to make them more explanatory?

Reply:
Could we update these to be strict comparisons, e.g.  `===` instead of `==`

Reply:
I have added some comments, but will am refactoring this further as part of the other PR.